ckan/templates/snippets/context/group.html

```diff
@@ -3,7 +3,7 @@
 <div class="context-info">
   {% if description != 'null' %}
     {% set description = description|striptags %}
-    <p class="about" title="{{ description }}">
+    <p class="about" aria-label="Group description">
```

Comment:
Don't use arial-label on static text. This attribute is designed for links/buttons/toggles.